azurelinuxagent/common/protocol/metadata_server_migration_util.py

```diff
@@ -55,20 +59,17 @@ def _cleanup_metadata_protocol_certificates():
     _ensure_file_removed(lib_directory, _LEGACY_METADATA_SERVER_TRANSPORT_CERT_FILE_NAME)
     _ensure_file_removed(lib_directory, _LEGACY_METADATA_SERVER_P7B_FILE_NAME)
 
-def _reset_firewall_rules(osutil):
+
+def _reset_firewall_rules():
     """
     Removes MetadataServer firewall rule so IMDS can be used. Enables
     WireServer firewall rule based on if firewall is configured to be on.
     """
-    osutil.remove_firewall(dst_ip=_KNOWN_METADATASERVER_IP, uid=os.getuid(), wait=osutil.get_firewall_will_wait())
-    if conf.enable_firewall():
```

Comment:
setting up the firewall is already done in the main and env threads; no need to do it here